extensions/1password7/src/list.tsx

```diff
@@ -112,6 +129,25 @@ function getIconForCategory(categoryUUID: string) {
   }
 }
 
+function PasswordListCategory(props: { onePasswordMetaItemsCategory: OnePasswordMetaItemsCategory }) {
+  const onePasswordMetaItemsCategory = props.onePasswordMetaItemsCategory;
+  console.log(onePasswordMetaItemsCategory)
```

Comment:
```suggestion
  console.log(onePasswordMetaItemsCategory)
```
Can we remove this, no need to log secret data to the console 😄

Reply:
Whoopsie! Good catch.
Debug code left there.

Removed!